RadixWallet/Features/AssetsFeature/Components/StakeUnitList/Components/LSUDetails+View.swift

```diff
@@ -11,10 +11,10 @@ extension LSUDetails.State {
 			),
 			thumbnailURL: stakeUnitResource.resource.metadata.iconURL,
 			validatorNameViewState: .init(with: validator),
-			redeemableTokenAmount: .init(.init(
+			redeemableTokenAmount: [.init(
 				id: stakeUnitResource.resource.resourceAddress, // FIXME: IS THIS CORRECT
```

Comment:
comment still relevant? has it not yet been resolved? :)

Reply:
:) I missed to delete several times.